pkg/secrethub/client.go

```diff
@@ -23,127 +21,100 @@ type Client interface {
 	Users() UserService
 }
 
-// Decrypter decrypts data, typically an account key.
-type Decrypter interface {
-	// Unwrap decrypts data, typically an account key.
-	Unwrap(ciphertext *api.EncryptedData) ([]byte, error)
-}
+var (
+	errClient = errio.Namespace("client")
+)
 
-// Encrypter encrypts data, typically an account key.
-type Encrypter interface {
-	// Wrap encrypts data, typically an account key.
-	Wrap(plaintext []byte) (*api.EncryptedData, error)
-}
+// Client is a client for the SecretHub HTTP API.
+type Client struct {
```

Comment:
If we make this the client type, I think we should make `ListDirAccounts` private.

Reply:
Then it should always have been private. But sure. No reason to export it.

Reply:
Yes, should have been. With this change it becomes a problem though, as it is now accessible outside the package.